packages/server/src/api/routes/tests/search.spec.ts

```diff
@@ -3470,5 +3472,70 @@ describe.each([
           ])
         })
       })
+
+    isSql &&
+      describe("SQL injection", () => {
+        const badStrings = [
+          "1; DROP TABLE test;",
```

Comment:
Should this table names not match the created ones? If the table "test" was actually deleted, would the test catch it?

Reply:
I considered that and decided it wasn't really necessary, because the query will fail as the table is missing, causing the test to fail all the same.

Reply:
I have addressed this, PTAL 🙏